packages/cli/src/db.ts

```diff
@@ -26,7 +26,7 @@ export const connectionState: ConnectionState = {
 };
 
 // Ping DB connection every 2 seconds
-let pingTimer: NodeJS.Timer | undefined;
+let pingTimer: NodeJS.Timeout | undefined;
```

Comment:
Maybe not here but later we should also rename all these mistyped names from `-timer` to `-timeout`.